systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1361,7 +1372,13 @@ private void testDockerImagesForKafkaMirrorMaker2(String clusterName, String clu
 
     @BeforeAll
     void setup(ExtensionContext extensionContext) {
+
+        // operation timeout is increased in order to wait long enough for manual rolling update
+        List<EnvVar> coEnvVars = new ArrayList<>();
+        coEnvVars.add(new EnvVar("STRIMZI_OPERATION_TIMEOUT_MS", "90000", null));
```

Comment:
It is already set in `withOperationTimeout` - so you should update it in the builder. Also, check all the `CO_OPERATION_TIMEOUT...` constants and pick some "higher" value instead of pasting magic value there